src/core_plugins/kibana/public/discover/components/field_chooser/lib/detail_views/string.html

```diff
@@ -44,6 +44,7 @@ <h5 ng-show="!field.details.error">Quick Count <kbn-info info="Top 5 values base
 
       <a
         ng-href="{{vizLocation(field)}}"
+        ng-if="field.aggregatable"
```

Comment:
That makes sense. How about a compromise? Can you just put the `isFieldVisualizable` method on the scope instead of in a separate service? At least this way the code becomes self-documenting.